src/coreclr/src/pal/src/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ endif(CLR_CMAKE_TARGET_OSX OR CLR_CMAKE_TARGET_FREEBSD)
 
 if(NOT DEFINED ENV{ROOTFS_DIR})
   include_directories(SYSTEM /usr/local/include)
+elseif (CLR_CMAKE_TARGET_FREEBSD)
+  include_directories(SYSTEM $ENV{ROOTFS_DIR}/usr/local/include)
```

Comment:
This is strange, the compiler should use ROOTFS_DIR as a base automatically.

Reply:
I think ROOTFS_DIR/usr/include is but ROOTFS_DIR/usr/local/include is not. This is because some dependencies are coming from extra package.

Reply:
If CMAKE_SYSROOT is set to ROOTFS_DIR, then (depending on compiler support for `-sysroot`, which Clang and GCC possess), RPATHs and find* commands use that to prefix the search paths. However, we still need to manually prepend the prefix for include_directories etc.